openshift/Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+pipeline {
+    agent none
+    environment {
+        // Pipeline-wide vars
+        name =  "wps-web"
+        ver = "0.0.1"
+        projTools = "auzhsi-tools"
+        projDev = "auzhsi-dev"
+        projProd = "auzhsi-prod"
+        suffix = "pr-${env.CHANGE_ID}"
+        rcToken="https://chat.pathfinder.gov.bc.ca/hooks/7A8RwSpZPDAqc6uXS/z8nRXX43oifYWELkwJKeLj4e86rtyfvtJdqdvjSE3XoK2sri"
+    }
+    options {
+        disableResume()
+    }
+    stages {
+        stage('Startup') {
+            agent { label 'build' }
+            steps {
+                script {
+                    def filesInThisCommitAsString = sh(
```

Comment:
Isn't there a way we could move this to a bash file, en depending on the exit code do the currentBuild.rawBuild.delete()?
I guess it might only save one line of code in the Jenkinsfile - so maybe not worth it?

Reply:
No, this is a great idea.  We should keep removing inherited Jenkins code and are on the right track.  I'll throw it in a new ticket.

Reply:
https://github.com/bcgov/wps/issues/156